nftserv/config.go

```diff
@@ -8,21 +8,29 @@ import (
 	"os"
 )
 
-type Config struct {
-	Host       string       `json:"host"`
-	Port       uint16       `json:"port"`
-	AssetsPath string       `json:"assetsPath"`
-	AssetsExt  string       `json:"assetsExt"`
-	CertFile   string       `json:"certFile"`
-	KeyFile    string       `json:"keyFile"`
-	Extra      ServerExtras `json:"extra"`
-}
+const defaultWhitelistedOrigin = "*"
 
-type ServerExtras struct {
-	WhitelistedOrigin    *string `json:"whitelistedOrigin"`
-	MaxTitleLength       *int    `json:"maxTitleLength"`
-	MaxDescriptionLength *int    `json:"maxDescriptionLength"`
-}
+type (
+	Config struct {
+		AssetsConfig AssetsConfig `json:"assetsConfig"`
+		ServerConfig ServerConfig `json:"serverConfig"`
+	}
+
+	AssetsConfig struct {
+		AssetsPath string `json:"assetsPath"`
+		AssetsExt  string `json:"assetsExt"`
```

Comment:
Since the containing type already contains `Asset`, I propose
```suggestion
		Path string `json:"path"`
		Ext  string `json:"ext"`
```
json needs to be adopted.

Reply:
Thanks for seeing this :see_no_evil: